extension/src/persistence/util.ts

```diff
@@ -1,9 +1,15 @@
 import { commands, Memento } from 'vscode'
 
-export const resetPersistedState = async (workspaceState: Memento) => {
+export const resetPersistedState = async (
```

Comment:
Updated our `resetPersistedState` to include the global state since it is a form of persisted state.

Reply:
[Q] Are we sure that all of the keys under here only relate to our extension?